tests/eas/load_tracking.py

```diff
@@ -1180,9 +1171,9 @@ class UnbalancedHierarchy(_PELTTaskGroupsTest):
                                        +-----+
                                        | "/" |
                                        +-----+
-                                       /     \
-                                      /       \
-                                  +------+   t0_1
+                                       /
```

Comment:
Since you are at modifying this comment... can you also remove all the double `/` that we use to represent connections?
... it would make the file shorted and more consistent with the tests @Elieva recently added.

Reply:
Done